executor_manager/services/sandbox/repository.py

```
@@ -0,0 +1,496 @@
+                "metadata": sandbox.metadata,
+            }
+
+            hash_key = f"{SESSION_HASH_PREFIX}{task_id}"
+            data = json.dumps(sandbox_info)
+            self.redis_client.hset(hash_key, SANDBOX_FIELD_NAME, data)
+            self.redis_client.expire(hash_key, self._config.timeout.redis_ttl)
+
+            # Update active sandboxes ZSet with current timestamp
+            self.redis_client.zadd(ACTIVE_SANDBOXES_ZSET, {str(task_id): time.time()})
+
+            return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)